### PR TITLE
MNT Replace sass-lint with stylelint

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Issues
- https://github.com/silverstripe/webpack-config/issues/69

### CODE REVIEW

File: .stylelintrc.yml
Comment (ChloeSartorelli) [inline .stylelintrc.yml:1]: 1. Move this all into a new repo (like https://github.com/silverstripe/eslint-config/)
2. Make sure all modules can share the same stylelint rules by referencing the new repo

Comment (ChloeSartorelli) [reply .stylelintrc.yml:1]: This will require updating all of the PRs so I won't continue reviewing until you've either responded or made that change.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: .stylelintrc.yml
Comment (sabina-talipova) [reply .stylelintrc.yml:1]: Done. Moved configuration to the `silverstripe/eslint-config` module.

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**